### PR TITLE
added todo list

### PR DESCRIPTION
**Pull request by abhishek-mullackal**
@AnandBaburajan  added a to-do component .please review.

### CODE REVIEW

File: todaythediary/src/components/todo/Form.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import { BsPlusCircleFill } from 'react-icons/bs';
+
+const Form = ({inputText,setInputText, todos, setTodos}) =>{
+
+    const inputTextHandler = (e) =>{
+        setInputText(e.target.value);
+    };
+    const submitTodoHandler =(e) =>{
+        e.preventDefault();
+        setTodos([...todos, {text:inputText , completed: false, id: Math.random()*10000}])
```
Comment (anandbaburajan) [inline todaythediary/src/components/todo/Form.js]: Can you explain why `Math.random()*10000` is used here for the ID instead of ordered numbers? Thanks!

**Review by anandbaburajan — COMMENTED** [review]
@abhishek-mullackal Thanks! The code and the working looks good to me! I've added a comment after which this can be merged! :D

File: todaythediary/src/components/todo/Form.js (hunk 1 above)
Comment (abhishek-mullackal) [reply todaythediary/src/components/todo/Form.js]: i have tried to implement unique id using uuid but failed :( . No problem. I will try again and update soon as possible .

File: todaythediary/src/components/todo/Form.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React,{useState} from 'react';
+import { BsPlusCircleFill } from 'react-icons/bs';
+
+const Form = ({inputText,setInputText, todos, setTodos}) =>{
+
+    const inputTextHandler = (e) =>{
+        setInputText(e.target.value);
+    };
+    const [counter,setCounter] = useState(0);
+    const incrementer = ()=>{
```
Comment (anandbaburajan) [inline todaythediary/src/components/todo/Form.js]: Renaming `counter` to `count`, `setCounter` to `setCount` and `incrementer` to `incrementCount` would be better. The new ID generation looks better!

Comment (abhishek-mullackal) [reply todaythediary/src/components/todo/Form.js]: Thank You.
I will do that soon.

**Comment by anandbaburajan** [comment]
Nice, merging!

**Pull request merged**